src/components/Devices/DeviceDetail.vue

```diff
@@ -1,123 +1,40 @@
-          </b-card>
-        </div>
-        <div v-else>
-          <b-table
-            :items="device.Users"
-            :fields="deviceUsersTableFields"
-            :sort-by="userSortBy"
-            striped
-            hover
-            outlined
-            responsive
-            data-cy="users-table"
-          >
-            <template v-slot:cell(admin)="data">
-              {{ data.item.DeviceUsers.admin ? "Yes" : "No" }}
-            </template>
-
-            <template v-slot:cell(controls)="data">
-              <b-modal
-                id="device-user-remove-self"
-                title="Remove yourself from device"
-                @ok="removeUser(data.item.username)"
-                ok-title="Remove"
-                ok-variant="danger"
-                v-model="showUserRemoveSelfModal"
-              >
-                <p>
-                  Are you sure you want to remove yourself from this device? You
-                  will no longer be able to view recordings from this device and
-                  you will not be able to add yourself back to the device.
-                </p>
-              </b-modal>
-              <b-button
-                v-b-tooltip.hover
-                v-if="isDeviceAdmin"
-                title="Remove user from device"
-                class="trash-button"
-                variant="light"
-                @click="removeUserCheckIfSelf(data.item.username, uiUser)"
-              >
-                <font-awesome-icon icon="trash" size="1x" />
-              </b-button>
-            </template>
-          </b-table>
-        </div>
-        <hr />
-        <div :class="'versions'">
-          <h2>Current software versions</h2>
-          <div v-if="!software.result">Message: {{ software.message }}</div>
-          <div
-            v-if="
-              software.result &&
-              software.result.EventDetail &&
-              software.result.EventDetail.details
-            "
-          >
-            <div
-              v-for="(version, component) in software.result.EventDetail
-                .details"
-              :key="component"
-            >
-              <b>{{ component }}</b
-              >: {{ version }}
-            </div>
-            <div v-if="software.result.dateTime">
-              <i
-                >Recorded {{ dayOfSnapshot.toLowerCase() }} at
-                {{ timeOfSnapshot }}</i
-              >
-            </div>
-            <p>
-              Current released software versions are listed
-              <a
-                href="https://github.com/TheCacophonyProject/saltops#branch-prod"
-                >here</a
-              >.
-            </p>
-          </div>
-        </div>
-      </b-col>
-    </b-row>
-  </div>
+  <b-container>
+    <b-tabs
+      card
+      class="device-tabs"
+      nav-class="device-tabs-container container"
+      v-model="currentTabIndex"
+    >
+      <b-tab title="Current Software">
```

Comment:
Should this and all events have TabTemplate also but just with no value in the badge?